lib/instances/GuiService.lua

```diff
@@ -25,4 +27,13 @@ end
 function GuiService.prototype:SafeZoneOffsetsChanged()
 end
 
+function GuiService.prototype:IsTenFootInterface()
+end
```

Comment:
This should probably return a bool like the real `IsTenFootInterface` does; we can just return `false` here until we have configurable habitats.